cmd/snap-confine/mount-support.c

```diff
@@ -177,27 +177,36 @@ static void sc_setup_mount_profiles(const char *snap_name)
 {
 	debug("%s: %s", __FUNCTION__, snap_name);
 
-	FILE *f __attribute__ ((cleanup(sc_cleanup_endmntent))) = NULL;
-	const char *mount_profile_dir = "/var/lib/snapd/mount";
-
+	FILE *desired __attribute__ ((cleanup(sc_cleanup_endmntent))) = NULL;
+	FILE *current __attribute__ ((cleanup(sc_cleanup_endmntent))) = NULL;
```

Comment:
Fwiw, I found "current" confusing as well but couldn't think of anything better when reviewing